modules/system/defaults/WindowManager.nix

```diff
@@ -56,11 +56,36 @@ with lib;
         Hide items in Stage Manager.
       '';
     };
+
+    system.defaults.WindowManager.EnableTilingByEdgeDrag = mkOption {
+      type = types.nullOr types.bool;
+      default = null;
+      description = ''
+        Enable dragging windows to screen edges to tile them.
+      '';
+    };
+
+    system.defaults.WindowManager.EnableTopTilingByEdgeDrag = mkOption {
+      type = types.nullOr types.bool;
+      default = null;
+      description = ''
+        Enable dragging windows to the menu bar to fill the screen.
+      '';
+    };
+
+    system.defaults.WindowManager.EnableTilingOptionAccelerator = mkOption {
+      type = types.nullOr types.bool;
+      default = null;
+      description = ''
+        Enable holding alt to tile windows.
+      '';
+    };
+
     system.defaults.WindowManager.EnableTiledWindowMargins = mkOption {
       type = types.nullOr types.bool;
       default = null;
       description = ''
-        Enable Window Margins. The default is true.
+        Enable window margins when tiling windows.
```

Comment:
```suggestion
        Enable window margins when tiling windows. The default is true.
```

Reply:
I can add this back if you'd like, but none of the other options in this file follow the convention of defaulting to null and listing the OS default so I dropped it from the description.

Reply:
Could you add it to all the options in this file?

Reply:
Sure, you'd like the system defaults added, not "null"?

Reply:
Added to the description but the value is `null`, this is the convention we generally follow

Reply:
Alright, done! I used `defaults delete` manually to make sure I had the correct values for the defaults in the description.